wifiphisher/common/tui.py

```diff
@@ -939,12 +940,16 @@ def display_info(self, screen, info):
                 screen.addstr(raw_num, 0, client)
                 raw_num += 1
         try:
+
             # print the dhcp lease section
             screen.addstr(7, 0, "DHCP Leases: ", self.blue_text)
-            if os.path.isfile('/var/lib/misc/dnsmasq.leases'):
-                dnsmasq_output = check_output(
-                    ['head', '-5', '/var/lib/misc/dnsmasq.leases'])
-                screen.addstr(8, 0, dnsmasq_output)
+            victims_instance = victim.Victims.getInstance()
+            dict_string = ""
+            for key in victims_instance.victims_dic.keys():
+                dict_string = key + "\t" + victims_instance.victims_dic[key].ip_address + "\t" + victims_instance.victims_dic[key].hostname + "\t" + victims_instance.victims_dic[key].vendor
+                dict_string = dict_string + "\n"
```

Comment:
Will this work even we don't have any victims' OS?

Reply:
Can we also output the MAC address somewhere? Also, note that you can use different colors for each victim's attribute.

Reply:
the key is mac, if we dont know it will print unkown, i can try with colours and see what it looks like